packages/x-date-pickers/src/MultiSectionDigitalClock/tests/MultiSectionDigitalClock.test.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import { expect } from 'chai';
+import { spy } from 'sinon';
+import {
+  MultiSectionDigitalClock,
+  MultiSectionDigitalClockProps,
+} from '@mui/x-date-pickers/MultiSectionDigitalClock';
+import { adapterToUse, createPickerRenderer } from 'test/utils/pickers-utils';
+import { multiSectionDigitalClockHandler } from 'test/utils/pickers/viewHandlers';
+
+describe('<MultiSectionDigitalClock />', () => {
+  const { render } = createPickerRenderer();
+
+  describe('Reference date', () => {
+    it('should use `referenceDate` when no value defined', () => {
+      const onChange = spy();
+
+      render(
+        <MultiSectionDigitalClock
+          onChange={onChange}
+          referenceDate={adapterToUse.date(new Date(2018, 0, 1, 12, 30))}
+        />,
+      );
+
+      multiSectionDigitalClockHandler.setViewValue(
+        adapterToUse,
+        adapterToUse.setMinutes(adapterToUse.setHours(adapterToUse.date(), 15), 30),
+      );
+      expect(onChange.callCount).to.equal(3);
+      expect(onChange.lastCall.firstArg).toEqualDateTime(new Date(2018, 0, 1, 15, 30));
+    });
+
+    it('should not use `referenceDate` when a value is defined', () => {
+      const onChange = spy();
+
+      function ControlledMultiSectionDigitalClock(props: MultiSectionDigitalClockProps<any>) {
```

Comment:
Yes
Because this one has several steps, so if I don't do the wrapper, the controlled value does not update, and the 2nd step is done with the initial value, not with the value created by the 1st step.